<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplitDialog</class>
 <widget class="QDialog" name="SplitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Split item</string>
  </property>
  <property name="windowIcon">
   <iconset theme="insert-link">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="classLabel">
     <property name="text">
      <string>Class</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="Keep">
     <property name="toolTip">
      <string>If enabled keeps oryginal objects and marks them as removed.</string>
     </property>
     <property name="text">
      <string>Keep changed objects</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="Hide">
     <property name="toolTip">
      <string>If enabled hides neartes connections to object.</string>
     </property>
     <property name="text">
      <string>Hide nearest connections</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar"/>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="procedButton">
       <property name="text">
        <string>Split</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="Class">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>&lt;p&gt;Classes of items to split items with them in geometry. Next values can be separated by comma sign.&lt;/p&gt;</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>111</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SplitDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>procedButton</sender>
   <signal>clicked()</signal>
   <receiver>SplitDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
